src/bioregistry/app/templates/collections.html

```diff
@@ -1,7 +1,7 @@
 {% extends "base.html" %}
 {% import "macros.html" as utils %}
 
-{% block title %}Bioregistry - Colletions{% endblock %}
+{% block title %}Bioregistry - Collections{% endblock %}
```

Comment:
It would be nice to add a sentence or two on this page about what a "collection" is.

Reply:
On that note, a simple sentence on top of the Registry and Metaregistry pages would also be nice to describe what those are directly on the page.